File: skin/src/main/java/com/donfyy/skin/SkinActivity.java
package com.donfyy.skin;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;

import com.donfyy.skinlib.SkinManager;

import androidx.annotation.Nullable;

public class SkinActivity extends Activity {


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_skin);
    }

    public void change(View view) {
        SkinManager.loadSkin("/data/data/com.donfyy.skin/skin/skin-debug.apk");
    }

    public void restore(View view) {
        SkinManager.loadSkin(null);
    }
}

File: skin/src/main/java/com/donfyy/skin/SkinActivity.kt
package com.donfyy.skin

import android.app.Activity
import android.os.Bundle
import android.view.View
import com.donfyy.skinlib.SkinManager.loadSkin

class SkinActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_skin)
    }

    fun change(view: View?) {
        loadSkin("/data/data/com.donfyy.skin/skin/skin2-debug.apk")
        //        SkinManager.loadSkin("/data/data/com.donfyy.skin/skin/skin-debug.apk");
    }

    fun restore(view: View?) {
        loadSkin(null)
    }
}